aws-datastore/src/test/java/com/amplifyframework/datastore/storage/sqlite/SQLCommandProcessorTest.java

```diff
@@ -166,4 +167,15 @@ public void executeExistsReturnsFalseWhenItemDoesntExist() throws AmplifyExcepti
         SqlCommand existsCommand = sqlCommandFactory.existsFor(blogOwnerSchema, predicate);
         assertFalse(sqlCommandProcessor.executeExists(existsCommand));
     }
+
+    /**
+     * Create a BlogOwner, but don't insert it.  Then verify that executeExists returns false.
+     * @throws AmplifyException on failure to create ModelSchema from class.
+     */
+    @Test
+    public void testIndexNotCreatedWhenFieldsInBelongsTo() throws AmplifyException {
```

Comment:
Do you have adequate testing for this?  I only see that it’s testing that one index is created, but it doesn’t check the contents of that index.

Are there any other edge cases that should be checked?

Also the comment says `Then verify that executeExists returns false` but I don’t see that in the test.

Reply:
Resolved.